res/README.md

```diff
@@ -0,0 +1,6 @@
+# Windows Terminal and Console Assets
+
+The assets in this directory do not fall under the same [license](https://raw.githubusercontent.com/microsoft/terminal/master/LICENSE) as the rest
+of the Windows Terminal code.
+
+Please consule the [license](./LICENSE) in this directory for applicable terms.

```

Comment:
Frick. It's okay, this pull request is broken anyway because the WAP packaging project is a huge house of cards and not the alright kind like the first two seasons of the critically-acclaimed show of the same name.